docs/dbus/manager.xml

```diff
@@ -1056,6 +1056,394 @@
+      <!-- id type: "uuid" or "name" -->
+      <arg name="id_type" type="s" direction="in" />
+      <!-- Object path of pool to be stopped. -->
+      <arg name="pool" type="o" direction="in" />
+      <!--
+        b: True if the pool was stopped.
+        s: UUID of the stopped pool.
+      -->
+      <arg name="result" type="(bs)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Unset a key in the kernel keyring. -->
+    <method name="UnsetKey">
+      <!-- Key description of the key to unset. -->
+      <arg name="key_desc" type="s" direction="in" />
+      <!-- b: True if the key was in the keyring, otherwise false. -->
+      <arg name="result" type="b" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!--
+      a{sa{sv}}: Mapping of Stratis pool UUIDs to information about the
+	devices associated with the UUID. These sets of devices may comprise
+        a Stratis pool that can be started with the StartPool message.
+        s: Stratis pool UUIDs
+	    a{sv}: key/value mapping
+          s: keys: possible values are "devs", "key_description", "clevis_info"
+          v: Values for the keys:
+	    "devs": always present, an array of mappings of device information
+	      Contains two keys: "uuid" and "devnode"
+	    "clevis_info": optional, not present if pool is unencrypted.
+	      Only used with V1 pools.
+	      (bv): information about Clevis configuration for all devices
+	        if b is true:
+	          devices have consistent Clevis information
+	          v = (bs): Clevis configuration information for all devices
+	            b: if false, devices are not encrypted with Clevis
+	            s: Clevis configuration
+	        else:
+	          devices have inconsistent Clevis configurations
+	          v = s: error message explaining inconsistency
+	    "key_description": optional, not present if pool is unencrypted.
+	      Only used with V1 pools.
+	      (bv): information about kernel key encryption for all devices
+	        if b is true:
+	          devices have consistent key information
+	          v = (bs): key information for all devices
+	            b: if false, devices are not encrypted with a key
+	            s: key description
+	        else:
+	          devices have inconsistent key information
+	          v = s: error message explaining inconsistency
+	    "metadata_version": optional, only present for V2 pools
+	      bt: optional metadata version number
+	        b: true if the metadata version is consistent for all devices
+	        t: the version number
+	    "features": optional map of pool features, only present for V2 pools
+	      b{sb}: optional map of features to values
+	        b: true if the feature information is consistent for all devices
```

Comment:
I don't think this is correct. I looked into the code and the reason it's optional seems to be the case where no metadata is found on the device. It's at the pool level metadata level and not something we determine across multiple encryption devices, so there's not really a way for it to be inconsistent across devices.